.github/workflows/ci.yml

```diff
@@ -3,49 +3,112 @@ name: CI
       - name: install dependencies
+        run: sudo apt-get install libevent-dev libboost-system-dev libboost-filesystem-dev libboost-test-dev libboost-thread-dev libminiupnpc-dev libzmq3-dev libdb4.8-dev libdb4.8++-dev
+      - name: install alt-integration-cpp
+        run: |
+          (
+          cd /opt;
+          wget https://github.com/VeriBlock/alt-integration-cpp/archive/${ALTINTEGRATION_VERSION}.tar.gz;
+          tar -xf ${ALTINTEGRATION_VERSION}.tar.gz;
+          cd alt-integration-cpp-${ALTINTEGRATION_VERSION};
+          mkdir build;
+          cd build;
+          cmake .. -DCMAKE_BUILD_TYPE=Release -DTESTING=OFF -DWITH_ROCKSDB=OFF;
+          sudo make -j2 install
+          )
+      - name: autogen
+        run: ./autogen.sh
+      - name: configure
+        run: ./configure
+          --without-gui
+          --enable-werror
+          --disable-bench
+          --disable-gmock
+      - name: make
+        run: make -j2
+
+  x86_64-w64-mingw32:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+        with:
+          path: main
+      - name: Prepare ccache timestamp
+        id: cache_timestamp
         run: |
-          cd depends
-          make HOST=${{ matrix.host }} -j
+          current_date=`date --utc +%Y%m%d_%H%M%SZ`
+          echo "::set-output name=timestamp::${current_date}"
+      - name: depends cache files
+        uses: actions/cache@v1.1.0
+        with:
+          path: ~/.depends_cache
+          key: mingw32-depends-${{ steps.cache_timestamp.outputs.timestamp }}
+          restore-keys: |
+            mingw32-depends-
+      - name: ccache cache files
+        uses: actions/cache@v1.1.0
+        with:
+          path: ~/.ccache
+          key: mingw32-ccache-${{ steps.cache_timestamp.outputs.timestamp }}
+          restore-keys: |
+            mingw32-ccache-
+      - name: install build requirements
+        run: sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl git g++-mingw-w64-x86-64 ccache
+      - name: set the default mingw32 g++ compiler option to posix
+        run: sudo update-alternatives --set x86_64-w64-mingw32-g++ /usr/bin/x86_64-w64-mingw32-g++-posix
+      - name: strip out problematic Windows %PATH% imported var
+        run: PATH=$(echo "$PATH" | sed -e 's/:\/mnt.*//g')
+      - name: set host variable and build dependencies for x86_64-w64
+        env:
+          CC: x86_64-w64-mingw32-gcc
+          CXX: x86_64-w64-mingw32-g++
+          HOST: x86_64-w64-mingw32
+          BASE_CACHE: ~/.depends_cache
+        run: cd depends && make -j2 NO_QT=1
       - name: autogen
         run: ./autogen.sh
       - name: configure
-        run: CONFIG_SITE=$PWD/depends/${{ matrix.host }}/share/config.site ./configure
+        run: CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure --prefix=/
```

Comment:
```suggestion
        run: CONFIG_SITE=$PWD/depends/$HOST/share/config.site ./configure --prefix=/
```

Reply:
also, is it **really** necessary to set prefix=/ ?

Reply:
I feel that we should stick closely to the BTC core instructions which include setting the --prefix=/.

![image](https://user-images.githubusercontent.com/35861207/79747778-ea5d9a00-82d1-11ea-8be8-d23e5e8554ce.png)